tailwind.config.js

```diff
@@ -1,12 +1,19 @@
 /** @type {import('tailwindcss').Config} */
 export default {
-  content: [
-    "./index.html",
-    "./src/**/*.{js,ts,jsx,tsx}",
-  ],
+  content: ["./index.html", "./src/**/*.{js,ts,jsx,tsx}"],
   theme: {
-    extend: {},
+    extend: {
+      backgroundImage: {
+        "tick-mark": "url('./assets/images/TickMark.svg')",
```

Comment:
I like this approach. Also, I would like you think about this approach in scale. 
I personally prefer to use inline svg element. It's easy to maintain the svg in future as it will be scoped inside the component. 
Just a thought... 🙂